pyproject.toml

```diff
@@ -38,7 +38,7 @@ dependencies = [
     "pydantic~=2.0",
     "jinja2",
     "magicattr~=0.1.6",
-    "litellm",
+    "litellm==1.55.3",
```

Comment:
LiteLLM version 1.55.3 is the only version that correctly supports passing `retry_policy` to `completion()` while respecting the number of retries specified by the policy